components/script/dom/document.rs

```diff
@@ -765,7 +786,56 @@ impl Document {
 
         if let MouseEventType::Click = mouse_event_type {
             self.commit_focus_transaction(FocusType::Element);
+
+            // https://w3c.github.io/uievents/#event-type-dblclick
+            let DBL_CLICK_TIMEOUT = Duration::from_millis(PREFS.get("dom.document.dblclick_timeout").as_u64().unwrap());
+            let DBL_CLICK_DIST_THRESHOLD = PREFS.get("dom.document.dblclick_dist").as_u64().unwrap();
```

Comment:
These are constants and shouldn't always be recomputed every time there's a click event.